src/main/java/uk/gov/hmcts/reform/draftstore/config/SwaggerConfiguration.java

```diff
@@ -17,7 +17,7 @@
     public Docket api() {
         return new Docket(DocumentationType.SWAGGER_2)
                 .select()
-                .apis(RequestHandlerSelectors.basePackage(DraftStoreApplication.BASE_PACKAGE_NAME + ".controllers"))
+                .apis(RequestHandlerSelectors.basePackage(DraftStoreApplication.BASE_PACKAGE_NAME + ".endpoint.v2"))
```

Comment:
would `endpoint` be a better base package?

Reply:
view description of this PR. excluding v3 for now.